static/app/components/panels/panelTable.tsx

```diff
@@ -71,52 +72,58 @@ export type PanelTableProps = {
  *       with `headers`. Then we can get rid of that gross `> *` selector
  * - [ ] Allow customization of wrappers (Header and body cells if added)
  */
-const PanelTable = ({
-  headers,
-  children,
-  isLoading,
-  isEmpty,
-  disablePadding,
-  className,
-  emptyMessage = t('There are no items to display'),
-  emptyAction,
-  loader,
-  stickyHeaders = false,
-  ...props
-}: PanelTableProps) => {
-  const shouldShowLoading = isLoading === true;
-  const shouldShowEmptyMessage = !shouldShowLoading && isEmpty;
-  const shouldShowContent = !shouldShowLoading && !shouldShowEmptyMessage;
-
-  return (
-    <Wrapper
-      columns={headers.length}
-      disablePadding={disablePadding}
-      className={className}
-      hasRows={shouldShowContent}
-      {...props}
-    >
-      {headers.map((header, i) => (
-        <PanelTableHeader key={i} sticky={stickyHeaders}>
-          {header}
-        </PanelTableHeader>
-      ))}
-
-      {shouldShowLoading && (
-        <LoadingWrapper>{loader || <LoadingIndicator />}</LoadingWrapper>
-      )}
-
-      {shouldShowEmptyMessage && (
-        <TableEmptyStateWarning>
-          <p>{emptyMessage}</p>
-          {emptyAction}
-        </TableEmptyStateWarning>
-      )}
-
-      {shouldShowContent && getContent(children)}
-    </Wrapper>
-  );
-};
+const PanelTable = forwardRef(
+  (
+    {
```

Comment:
I would handle this at the export level. It would remove a level of nesting and also give the exported component a name as well.